db/migrate/20160223172116_rolify_create_roles.rb

```diff
@@ -0,0 +1,19 @@
+class RolifyCreateRoles < ActiveRecord::Migration
+  def change
+    create_table(:roles) do |t|
+      t.string :name
```

Comment:
I don't usually like to modify migrations generated by gems, for all I know some part of their behavior is reliant on these constraints not existing at the database level. I can attempt to add them in another migration and see if anything breaks, but I didn't want to make any assumptions about the implementation here.